From 2650e2e6d7878a4fdb908caf3fdfb8641f406692 Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Tue, 4 Jun 2013 13:57:18 +0800
Subject: [PATCH 204/602] ENGR00265367 fix system hang when access browser

Revert "ENGR00263121 enable vg acceleration for android dual display"

This reverts commit f94e3456a3a14cb92205675efa26b6e854c93235.

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 hdmidongle_6dq/init.rc | 2 +-
 sabreauto_6q/init.rc   | 2 +-
 sabresd_6dq/init.rc    | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/hdmidongle_6dq/init.rc b/hdmidongle_6dq/init.rc
index 801438a..a52334a 100755
--- a/hdmidongle_6dq/init.rc
+++ b/hdmidongle_6dq/init.rc
@@ -33,7 +33,7 @@ on boot
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
 
-    setprop hwc.stretch.filter  0
+    setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
 
     # fsl omx graphic manager media framework property
diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index 4227af9..da26f1b 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -33,7 +33,7 @@ on boot
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
 
-    setprop hwc.stretch.filter  0
+    setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
 
     # fsl omx graphic manager media framework property
diff --git a/sabresd_6dq/init.rc b/sabresd_6dq/init.rc
index fd9d1e0..5f105d1 100755
--- a/sabresd_6dq/init.rc
+++ b/sabresd_6dq/init.rc
@@ -38,7 +38,7 @@ on boot
     setprop debug.sf.enable_hgl 1
     setprop debug.egl.hw	1
 
-    setprop hwc.stretch.filter  0
+    setprop hwc.stretch.filter  1
     setprop hwc.enable_dither   1
 
     # fsl omx graphic manager media framework property
-- 
1.8.0

